src/services/inline-context/helpers.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+/**
+ * Copyright (c) Garuda Labs, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+export const LINE_BREAK = '\n';
+export const EMPTY = '';
+export const SPACE = ' ';
+
+export const ignoreSpacesAfterLineBreak = (input: string): string =>
+  input.replace(/\n\s+/g, LINE_BREAK);
+
+export const convertLineBreaksIntoSpaces = (input: string): string =>
+  input.replace(/\n/g, SPACE);
+
+export const ignoreSpacesFollowingSpace = (input: string[]): string[] => {
+  return input.map((value: string, index: number, values: string[]): string => {
+    if (index > 0) {
+      const previousEntry = values[index - 1];
+      const lastChar = previousEntry[previousEntry.length - 1];
```

Comment:
if `previousEntry` is empty, won't this be an invalid index?

Reply:
fixed in [f1aff6b](https://github.com/Instawork/hyperview/pull/448/commits/f1aff6b31e45855558f1cba2d9837ae462db896b)